src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressionPatchFilterTest.java

```diff
@@ -180,36 +180,242 @@ private void shouldRejectLine(SuppressionPatchFilter filter, int lineNo, String
         return createSuppressionPatchFilter(fileName, "changed");
     }
 
-    @Ignore
     @Test
+    @Ignore("wait until testByConfig can process a dir contains no more than one file")
```

Comment:
please make it like `until http://...link.. when testByconfig .....`

the same for other tests.

Reply:
done